public/app/core/navigation/GrafanaRouteLoading.tsx

```diff
@@ -1,24 +1,42 @@
-import { css } from '@emotion/css';
+import { css, cx } from '@emotion/css';
 import React from 'react';
 
-import { LoadingPlaceholder, useStyles2 } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { useStyles2 } from '@grafana/ui';
+
+import { t } from '../internationalization';
 
 export function GrafanaRouteLoading() {
   const styles = useStyles2(getStyles);
 
   return (
-    <div className={styles.loadingPage}>
-      <LoadingPlaceholder text={'Loading...'} />
+    <div
```

Comment:
Could not be better to have the 'bouncing logo' as a component?

Reply:
Since it shouldn't get used outside of a loading context, I think this component is fine as it is. Maybe it isn't specific to route loading, so the name might be a change (?).

Reply:
extracted it out anyway, could be useful 👍